drivers/ioexpander/gpio.c

```diff
@@ -691,7 +691,7 @@ int gpio_pin_unregister(FAR struct gpio_dev_s *dev, int minor)
 int gpio_pin_unregister_byname(FAR struct gpio_dev_s *dev,
                                FAR const char *pinname)
 {
-  char devname[32];
+  char devname[64];
```

Comment:
I do not see how this should help. Maybe `snprintf(devname, sizeof(devname) - 5, "/dev/%s", pinname);` instead or `snprintf(devname, sizeof(devname) - sizeof("/dev/") - 1, "/dev/%s", pinname);`?

Reply:
You are right, the problem is fixed here: https://github.com/apache/nuttx/pull/8658. Let's close this PR.